forms-flow-api/src/formsflow_api/resources/filter.py

```diff
@@ -0,0 +1,168 @@
+"""API endpoints for filter resource."""
```

Comment:
Need to secure all these endpoints for `formsflow-reviewer` role.

Reply:
Filter creation access can be granted to a reviewer admin ? or designer ?

Reply:
I was thinking of letting the reviewer create the filter right on the tasks page.

Reply:
> Need to secure all these endpoints for `formsflow-reviewer` role.

done